### PR TITLE
[add] task-8 and task-9:  -done and -tested with care "don't be an ass"

### PR DESCRIPTION
**Pull request by hamzahdili2001**
I change some of your code in `file_storage.py` take a good look at it, changes I made it more dynamic and easy to understand 

**Still have a bug in the console.py  line:23** I can find the issue you can check it out before merging.

### CODE REVIEW

File: console.py [hunk 1]
```diff
@@ -19,6 +20,7 @@ def __init__(self) -> None:
         cmd.Cmd.__init__(self)
         self.classes = {
             "BaseModel": self.create_BaseModel,
+            "User": self.create_User,  # BUG: I don't know what is going on.
```
Comment (MrSakine) [inline console.py]: what bug ?

**Comment by MrSakine** [comment]
> I change some of your code in `file_storage.py` take a good look at it, changes I made it more dynamic and easy to understand
> 
> **Still have a bug in the console.py line:23** I can find the issue you can check it out before merging.

i'm checking, about that bug

File: console.py (hunk 1 above)
Comment (MrSakine) [reply console.py]: found it, you have forgot to create the method `create_User`

Comment (hamzahdili2001) [reply console.py]: SOLVED

**Review by MrSakine — APPROVED** [review]

**Pull request merged**